design-documents/graph-ql/coverage/product-reviews.graphqls

```diff
@@ -65,12 +68,17 @@ type CreateProductReviewOutput {
 input CreateProductReviewInput {
     sku: String! @doc(description: "The SKU of the product that the review is assigned")
     nickname: String! @doc(description: "The customer's nickname. Defaults to customer name if logged in.")
-    summary: String! @doc(description: "The review summary (a.k.a title")
-    text: String! @doc(description: "The review text.")
-    ratings: [ProductReviewRatingInput!]! @doc(description: "Ratings details by category. e.g price: 5, quality: 4 etc")
+    title: String! @doc(description: "The review title")
+    review_text: String! @doc(description: "The review text.")
```

Comment:
what the necessity of this change? smaller name informative enough I would avoid using snake_case fields if possible because of the nature of main graphql clients - js apps.

Reply:
1. Please see point 5 from PR description.
2. If the goal is to avoid snake_case fields please see point 6 also.

Reply:
We are designing these API from scratch, there is no need to be consistent with existing service contracts. 
We definitely need to make changes that make API more self-explanatory. 

I believe current naming of the field `ProductReview.text` is intuitive and avoids repetitions compared to `ProductReview.review_text`.
`ProductReview.title` vs `ProductReview.summary` is pretty subjective. When we were selecting name for this field we chose `summary` because it it not an actual review title, but rather a short summary.